.github/workflows/fuzz.yaml

```diff
@@ -0,0 +1,32 @@
+name: fuzz
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  fuzz:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        if: github.event_type == 'pull_request'
+
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "^1.17"
+
+      - name: Fuzz
+        run: go test -run ^TestFuzz$ cdr.dev/enterprise-helm/tests -count=1000
```

Comment:
I'm not using any fuzzing libraries so I don't think this would integrate all that well. I wasn't too sure how to map the random bytes `go-fuzz` gives us into a `*CoderValues` struct, so I just used this faker package for now. Maybe once 1.18 comes out there will be faker packages that integrate into the built-in fuzzing.